docs/src/content/docs/explainers/async-transform.mdx

```diff
@@ -1,6 +1,274 @@
 ---
 title: Async Transform
-description: How to setup the Async Transform
+description: How the Async transform works
 ---
 
-This is a guide about how to setup the Async Transform!
+import { LinkCard, Tabs, TabItem, Aside, Code } from '@astrojs/starlight/components';
+import BlankLink from '@components/BlankLink.astro';
+
+This explainer is meant to show you how the `Async Transform` Vite plugin works so that you can be
+confident in using it. If you want to know how to setup the Vite plugin please go to the following
+link.
+
+<LinkCard
+	href="getting-started/installation/#setup-the-async-transform"
+	title="Setup the Async Transform"
+	description="Read the short guide about how to setup the Async Transform"
+/>
+
+## What's a Vite plugin
+
+For those who doesn't know anything about what a Vite plugin even is and want to know more here's a
```

Comment:
When I read "you" in a doc, I am under the impression the doc talks to me rather than talking to a fictive audience. So I wouldn't use phrasing that suppose the presence of other people reading this with me.

```suggestion
If you don't know anything about what a Vite plugin even is and want to know more, here's a
```